SystemServices/SystemServices.h

```diff
@@ -51,6 +52,7 @@
 #define EVT_ONTEMPERATURETHRESHOLDCHANGED "onTemperatureThresholdChanged"
 #define EVT_ONMACADDRESSRETRIEVED         "onMacAddressesRetreived"
 #define EVT_ONREBOOTREQUEST               "onRebootRequest"
+#define EVT_ONFWPENDINGREBOOT             "onFwPendingReboot" /* Auto Reboot notifier */
```

Comment:
Change Thunder event name to onFirmwarePendingReboot

Reply:
Done